### PR TITLE
Prepare DAML-LF 1.8 release

### PR DESCRIPTION
**Pull request by cocreature**
We will only include type synonyms and package metadata (which are
used for Cross-SDK model upgrades). Everything else stays in 1.dev for
now.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs [hunk 1]
```diff
@@ -31,6 +31,11 @@ version1_6 = V1 $ PointStable 6
 version1_7 :: Version
 version1_7 = V1 $ PointStable 7
 
+-- | DAML-LF version 1.8
+-- TODO (MK) Change once LF 1.8 is frozen.
+version1_8 :: Version
+version1_8 = V1 $ PointStable 8
```
Comment (remyhaemmerle-da) [inline compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs]: Are you sure it is not 
```versionDev = V1 PointDev``` ?

Comment (cocreature) [reply compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs]: good catch, fixed!

**Pull request merged**